spacecmd/tests/test_kickstart.py

```diff
@@ -374,9 +374,9 @@ def test_kickstart_delete_invalid_profile(self, shell):
         assert_args_expect(shell.do_kickstart_list.call_args_list,
                            [(('', True), {})])
 
-    def test_kickstart_delete_some_invalid_profile(self, shell):
```

Comment:
Shouldn't we keep this test?

Reply:
No, this one tests for the case I removed in the code. Basically, if **some** of the args are not found, it used to throw an error, which didn't work with wildcards. Now it works with only the args it could match. The removed part doesn't exist in any other methods that deal with the globs in the same way, so I removed it from this as well.

Reply:
How does the user know that some of the passed profiles were not found after your change?